commons-api/src/main/java/org/exoplatform/commons/api/notification/plugin/NotificationPluginUtils.java

```diff
@@ -41,6 +43,9 @@ public class NotificationPluginUtils {
   
   public static final String BRANDING_COMPANY_NAME_SETTING_KEY = "exo.branding.company.name";
 
+  private static final Pattern EMAIL_PATTERN = Pattern.compile("^[_a-zA-Z0-9-+]+(\\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)*(\\.[a-zA-Z]{2,})$");
```

Comment:
Could you use the same pattern variable ? org.exoplatform.commons.notification.NotificationUtils#EMAIL_PATTERN

Reply:
in this case, we need to add a dependency on the component-common which is not possible since the component-common  depends on commons-api (the current one)

Reply:
You can move the whole file under org.exoplatform.commons.api.notification package

Reply:
Or at least move the constants inside a File in that package, since moving the file may cause issues if it used in other modules/addons